src/cb_admin/pages/VisitorDetails.js

```diff
@@ -104,19 +104,12 @@ export default class VisitorDetailsPage extends React.Component {
   }
 
   componentDidMount() {
-    Visitors.genders()
+    CommunityBusiness.update() // used to check cookie permissions
+      .then(Visitors.genders)
```

Comment:
Needs to be a function

Reply:
Oh, oops, I've just noticed that this is a function

Reply:
but it *is* a function...